shapely/io.py

```diff
@@ -45,10 +45,7 @@ def to_wkt(
 
     The following limitations apply to WKT serialization:
 
-    - for GEOS <= 3.8 a multipoint with an empty sub-geometry will raise an exception
-    - for GEOS <= 3.8 empty geometries are always serialized to 2D
-    - for GEOS >= 3.9 only simple empty geometries can be 3D, collections are still
-      always 2D
+    - only simple empty geometries can be 3D, collections are always 2D
```

Comment:
```suggestion
    - only simple empty geometries can be 3D, empty collections are always 2D
```

(just to be a bit more explicit)